src/routes/cartRouter.ts

```diff
@@ -28,5 +28,7 @@ router.get(
   isCartIdExist,
   cartControllers.buyerGetCart
 );
+router.get("/user-get-order-status/:id",userAuthorization(["buyer"]), validation(orderStatusSchema), cartControllers.buyerGetOrderStatus )
```

Comment:
Again, I have seen you are sending the `**id** ` into the body request at the same time sending the **`id`** into params, this is wrong logic.